lib/core/src/Ouroboros/Network/Client/Wallet.hs

```diff
@@ -222,7 +224,7 @@ chainSyncFollowTip toCardanoEra onTipUpdate =
 -- callback.
 --
 -- See also 'send' for invoking commands.
-data ChainSyncCmd block (m :: * -> *)
+data ChainSyncCmd block (m :: Type -> Type)
```

Comment:
Starting to wonder if enabling `{-# LANGUAGE StarIsType #-}` as a default extension wouldn't have made your life easier here :sweat_smile: ... I guess going over this change is now for the better.